genie-core/src/main/java/com/netflix/genie/core/services/impl/LocalJobKillServiceImpl.java

```diff
@@ -57,6 +61,8 @@
     private final Executor executor;
     private final boolean runAsUser;
     private final ApplicationEventPublisher eventPublisher;
+    private final File baseWorkingDir;
+    private final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Also the bean is not present during tests, same as `jobDir` below. Let's take a look together. You can probably show me a trick or two to debug beans creation and injection.